packages/appsync-typegen-plugin/src/plugin.ts

```diff
@@ -0,0 +1,22 @@
+import { PluginFunction, Types } from '@graphql-codegen/plugin-helpers';
+import { GraphQLSchema } from 'graphql';
+import { RawAppSyncTypeConfig } from './visitors/appsync-visitor';
+
+import { generateSource as generateTypescriptSource } from './typescript';
+import { compileToLegacyIR } from './compiler/legacyIR';
+import { loadAndMergeQueryDocuments } from './loading';
+
+export const plugin: PluginFunction<RawAppSyncTypeConfig> = (
```

Comment:
This is the Typescript implementation of GraphQL code generator's interface - https://github.com/dotansimha/graphql-code-generator/blob/master/packages/utils/plugins-helpers/src/types.ts#L567